appendix_meta_rules.md

```diff
@@ -135,6 +178,7 @@ Like sigma rules , correlation rules have a title and a unique id to identify th
 **Attribute:** title
 
 A brief title for the rule that should contain what the rule is supposed to detect (max. 256 characters)
+<!-- shouldn't it be mandatory ?-->
```

Comment:
It is. That is defined in the schema. At least for the existing spec, it wasn't defined in the text body if a key is required if it's optional.